src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LogValuesFormatter.cs

```diff
@@ -128,6 +128,32 @@ private static int FindIndexOfAny(string format, char[] chars, int startIndex, i
         }
 
         public string Format(object?[]? values)
+        {
+            object?[]? formattedValues = values;
+
+            if (values != null)
+            {
+                for (int i = 0; i < values.Length; i++)
+                {
+                    object formattedValue = FormatArgument(values[i]);
+                    if (!ReferenceEquals(formattedValue, values[i]))
+                    {
+                        formattedValues = new object[values.Length];
+                        Array.Copy(values, 0, formattedValues, 0, i);
+                        formattedValues[i++] = formattedValue;
+                        for (; i < values.Length; i++)
+                        {
+                            formattedValues[i] = FormatArgument(values[i]);
+                        }
+                        break;
+                    }
+                }
+            }
+
+            return string.Format(CultureInfo.InvariantCulture, _format, formattedValues ?? Array.Empty<object>());
+        }
+
+        internal string FormatQuick(object?[]? values)
```

Comment:
>FormatQuick [](start = 24, length = 11)

I would call it something like FormatWithOverride or something similar

Reply:
Yes, thanks for the catch. I'll update my comment.

Reply:
Yes, I'll go with that! Thanks!